src/tests/routes/test_alerts.py

```diff
@@ -36,7 +36,7 @@
 ACCESS_TABLE = [
     {"id": 1, "group_id": 1, "login": "first_login", "hashed_password": "hashed_pwd", "scope": "user"},
     {"id": 2, "group_id": 1, "login": "second_login", "hashed_password": "hashed_pwd", "scope": "admin"},
-    {"id": 3, "group_id": 2, "login": "third_login", "hashed_password": "hashed_pwd", "scope": "device"},
+    {"id": 3, "group_id": 1, "login": "third_login", "hashed_password": "hashed_pwd", "scope": "device"},
```

Comment:
mind adding a comment when modifying a test case? (hard to review without reading again all the unittests :sweat_smile: )

Reply:
Ok, here it was to have one device in the group_id 1 (otherwise we have not alert linked to group 1)

Reply:
alright, and did the previous test case output change? I'm trying to check why we can't keep both / if it's useful

Reply:
Previously the group_id was not used so I just added group_id:1 to all accesses object just so that the field is not empty. No Utest was using it (in test_alerts at least)